apps/functions/deadline-submissions/deadline-submission-command/back-office-api-client.js

```diff
@@ -8,14 +8,13 @@ import config from './config.js';
  * @returns {Promise<number | null>}
  * */
 async function getCaseID(caseReference) {
-	const response = await got.get(
-		`https://${config.apiHost}/applications?reference=${caseReference}`
-	);
-	if (!response.ok) {
-		return null;
+	try {
+		return await got
+			.get(`https://${config.apiHost}/applications?reference=${caseReference}`)
+			.json();
```

Comment:
This won't return the just the ID now will it? Previously it returned `.id`

Reply:
Ah good spot. I've pushed a fix.